geode-core/src/main/java/org/apache/geode/internal/cache/BucketAdvisor.java

```diff
@@ -351,6 +351,23 @@ void removeBucket() {
     setHosting(false);
   }
 
+  /**
+   * Return all members hosting this bucket
+   *
+   * @return members hosting bucket
+   */
+  public Set<InternalDistributedMember> getAllMembers() {
```

Comment:
`getAllMembers` is kind of ambiguous. I recommend something more explicit such as `getAllHostingMembers`.